api_core/google/api_core/grpc_helpers.py

```diff
@@ -140,8 +140,11 @@ def create_channel(target, credentials=None, scopes=None, **kwargs):
         credentials, request, target, **kwargs)
 
 
-_ChannelCall = collections.namedtuple(
-    '_ChannelCall', ['request', 'timeout', 'metadata', 'credentials'])
+_MethodCall = collections.namedtuple(
+    '_MethodCall', ['request', 'timeout', 'metadata', 'credentials'])
```

Comment:
`list` here and `tuple` below. Pick one?

Reply:
Fixed.